languages/javascript/exercises/concept/closures/.docs/introduction.md

```diff
@@ -0,0 +1,32 @@
+**Closures** are a programming pattern in JavaScript which allows variables from an outer lexical context to be used inside of a nested block of code. JavaScript supports closures transparently, and they are often used without knowing what they are.
```

Comment:
I'm wondering if students will understand what "lexical context" means. The example below uses scope and the instructions also used "block of code", both of which might be easier to understand for a student.

Reply:
I'll add a sentence to describe it.  I found lexical context to be a common phrase when reading through resources for this.

Reply:
Cool

Reply:
Yes, lexical context is a technical term, but also the correct term!